index.html

```diff
@@ -635,15 +635,11 @@ <h3>Information Model</h3>
             <p>
                 [[[#directory-class-diagram]]] shows the information model of the
                 <a>Thing Description Directory</a>.
```

Comment:
This is not true anymore w.r.t. to the new figure.

Reply:
The new figure shows the ontology of the registered things. It can be cited after introducing Enriched TD.

Reply:
I updated this to " shows the diagram of the <a>Thing Description Directory</a> ontology"